includes/modules/pages/checkout_success/header_php.php

```diff
@@ -103,6 +103,7 @@
   $products_query = $db->bindVars($products_query, ':ordersID', $orders_id, 'integer');
   $products = $db->Execute($products_query);
 
+  $notificationsArray = [];

```

Comment:
Since the array is already initialized on line 88, let's not add it again.
Perhaps move it up before the loop so that it's available if the loop doesn't get processed.

Reply:
i did not notice that.  not sure why that error would come up.  let me explore.

Reply:
ok. this error was fixed already.  my mistake.  will delete.